INFO(  ): summery




Dtype,NON-Null Count,Column,memory usage,column datatypes and total datatypes in the dataset.
In The End U Get NONE in the return because info() does not return anything


In [5]:
import pandas as pd
df=pd.read_csv(r"C:\Users\ashay\OneDrive\Desktop\SACRIFICES OF COMFORT\Machine-Learning\Speedrun For MidTerm Exam\.venv\Workable\titanic_tested.csv")
# print(df)
print(df.info())

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 418 entries, 0 to 417
Data columns (total 12 columns):
 #   Column       Non-Null Count  Dtype  
---  ------       --------------  -----  
 0   PassengerId  418 non-null    int64  
 1   Survived     418 non-null    int64  
 2   Pclass       418 non-null    int64  
 3   Name         418 non-null    object 
 4   Sex          418 non-null    object 
 5   Age          332 non-null    float64
 6   SibSp        418 non-null    int64  
 7   Parch        418 non-null    int64  
 8   Ticket       418 non-null    object 
 9   Fare         417 non-null    float64
 10  Cabin        91 non-null     object 
 11  Embarked     418 non-null    object 
dtypes: float64(2), int64(5), object(5)
memory usage: 39.3+ KB
None


In the cabin total entires are 91  and  age has 332 and the total enteries is 418 then in the cabin the 327 is null entries and the age has 86 enteries

In the dtype has object this mean that have string , or categorical data (name,city)

